service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -448,4 +448,10 @@ public String getComplianceDeadline() {
     private final String deprivationOfLiberty;
 
     private final ReturnApplication returnApplication;
+
+    public boolean allocatedJudgeExists() {
+        {
+            return allocatedJudge != null ? true : false;
```

Comment:
Don't think we need the ternary operation. Returning the allocatedJudge != null evaluates to a condition itself no?

Reply:
good spot